doc/manual/source_localization/forward.rst

```diff
@@ -180,7 +180,7 @@ and
 
 .. tabularcolumns:: |p{0.2\linewidth}|p{0.3\linewidth}|p{0.5\linewidth}|
 .. _CHDJDEDJ:
-.. table:: Coordinate transformations in FreeSurfer and MNE software packages. The symbols :math:`T_x` are defined in :ref:`CHDFFJIJ`. Note: mne_make_cor_set /mne_setup_mri prior to release 2.6 did not include transformations :math:`T_3`, :math:`T_4`, :math:`T_-`, and :math:`T_+` in the fif files produced.
+.. table:: Coordinate transformations in FreeSurfer and MNE software packages.
```

Comment:
The links on table headers don't seem to work very well (https://circle-artifacts.com/gh/mne-tools/mne-python/621/artifacts/0/home/ubuntu/mne-python/doc/_build/html/manual/source_localization/forward.html#chdjdedj)
Also the links seem wrong with these long titles. See if you agree with the changes once the circle is ready.